en_US/deploy/cluster/lb.md

```diff
@@ -1,31 +1,35 @@
-# Cluster load balancing
+# Configure Load Balancer
 
-Load Balancer (LB) will balance the load among multiple network components, optimize resource usage, and avoid system malfunctions caused by overload.-
-
-Though not a mandatory component, it could bring some obvious system benefits, for example, when an LB is configured in EMQX:
+Load Balancer (LB) will balance the load among multiple network components, optimize resource usage, and avoid system malfunctions caused by overload. LB is not a mandatory component in EMQX, but could bring some obvious system benefits, for example:
```

Comment:
```suggestion
Load Balancer (LB) balances the load among multiple network components and optimizes resource usage to avoid system malfunctions caused by overload. LB is not a mandatory component in EMQX, but it can bring some obvious system benefits, for example:
```